include/arpa/nameser.h

```diff
@@ -7,6 +7,7 @@ extern "C" {
 
 #include <stddef.h>
 #include <stdint.h>
+#include <stdchecked.h>
```

Comment:
Maybe it's a better option to use _Ptr, _Array_ptr, etc. This would remove the dependency on this stdchecked header.

Reply:
Thanks for the suggestion! Can I add this header to the `include` or `src/include` folder instead? This would remove the external dependency and allow me to use the shorter and more readable aliases.

Reply:
Yes, you may add to `src/include`. However, I would prefer that both the string and the network directories follow the same convention. Please make sure you talk to @emoham and agree on a convention for the conversion - either use the shortened names (like `ptr`) or use the longer names (like `_Ptr`) in both the directories. Personally, I prefer the longer names (like `_Ptr`). That's also what the `checkedc-convert` tool uses.

Reply:
@emoham and I have agreed to use the shortened aliases for both the string and network directories. I have added `stdchecked.h` to the `include` folder as well.

Reply:
Per the email discussion, I've replaced the shortened aliases with the longer ones, and removed dependency on `stdchecked.h`.